cumulus/README.md

```diff
@@ -161,26 +165,26 @@ cargo build --release --bin polkadot
 # Clone
 git clone https://github.com/paritytech/polkadot-sdk
 
-# Compile
+# Compile (under `/polkadot-sdk/cumulus/polkadot-parachain`)
```

Comment:
I would remove this and better add there:
```
cd ./cumulus/polkadot-parachain`
cargo build --release --bin polkadot-parachain
```

or just use from the top directory:
```
cargo build --release -p polkadot-parachain-bin
```

Reply:
Changed, thanks